.github/workflows/autopr.yaml

```diff
@@ -22,6 +22,8 @@ jobs:
       run: |
         git config --local user.email "action@github.com"
         git config --local user.name "GitHub Action"
+        cd host/3.0
+        ./verify-pipelines.sh
```

Comment:
Please verify this in a test pipeline run as I'm not sure that the following commands will not be run based on the exit status, you may need to check the exit status and exit on this line.

Reply:
Thank you for your suggestion. Successfully verified the pipeline behavior.  Sent you an image offline but also feel free to checkout this workflow
https://github.com/Azure/azure-functions-docker/actions/runs/1242979688